utils/train_utils.py

```diff
@@ -396,8 +412,6 @@ def save_train_params(train_config, fsdp_config, rank):
     train_config.dist_checkpoint_root_folder
     + "/"
     + train_config.dist_checkpoint_folder
-    + "-"
```

Comment:
Why this change?

Reply:
When passing full paths as the model's name it creates chaotic paths